tests/Gedmo/Timestampable/ChangeTest.php

```diff
@@ -132,9 +132,7 @@ public function getDateValue($meta, $field): ?\DateTime
     }
 }
 
-/**
- * @phpstan-extends AbstractTrackingListener<array, TimestampableAdapter>
- */
+/** @phpstan-extends AbstractTrackingListener<array, TimestampableAdapter> */
```

Comment:
This change seems unrelated to this PR.

Reply:
Yeah that's definitely from different iterations of messing with the generics and rebasing things 😅